batchflow/models/torch/base.py

```diff
@@ -146,19 +145,19 @@ class TorchModel:
 
         Examples:
 
-        - ``{'decay': 'exp'}``
-        - ``{'decay': ('StepLR', {'steps_size': 10000})}``
-        - ``{'decay': {'name': MyCustomDecay, 'decay_rate': .5}``
-
-    n_iters : int
-        Frequency of making step of learning rate decay.
+        - ``{'decay': {'name: 'exp', 'frequency': 5, 'first_iter': 6, 'last_iter': 20}}``
+        - ``{'decay': {'name': StepLR', 'steps_size': 10000, 'frequency': 5}}``
+        - ``{'decay': {'name': MyCustomDecay, 'decay_rate': .5, 'frequency': 15, 'first_iter': 400}``
+        - ``{'decay': {'name': MyCustomDecay, 'decay_rate': .5, 'frequency': 15, 'first_iter': 400}``
```

Comment:
duplicate line;
Add explanation that the last example is a `warmup` of the learning rate